src/dash-table/components/Highlight/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Remarkable } from 'remarkable';
+import LazyLoader from 'dash-table/LazyLoader';
+
+export default class Highlight {
```

Comment:
1. This is an indirection for the `Remarkable` library to use `Highlighlight.js`, not the other way around so the file's & class name are misleading
2. This is not a component, this code should live in another folder